.github/workflows/create-design-review-issue.yml

```diff
@@ -12,6 +12,6 @@ jobs:
     steps:
       - name: Create Design Review Issue
         run: |
-          wget -q -O - "https://raw.githubusercontent.com/$GITHUB_REPOSITORY/master/.github/workflows/create-design-review-issue.js" | node
+          wget -q -O - "https://raw.githubusercontent.com/ampproject/amphtml/main/.github/workflows/create-design-review-issue.js" | node
```

Comment:
I don't have a problem with it but for my own information: why avoid `$GITHUB_REPOSITORY`?

Reply:
Using the actual repo name makes it easier to search for during refactors of this sort. (I nearly missed updating this file.)

A secondary goal was to make it explicit that this workflow is only meant to be used by the `ampproject` repo and not by forks, although this could be done even more clearly as follows:

https://github.com/ampproject/amphtml/blob/e03dba1b1fe7ff1d4dfeb59ba257ab4d0d93341f/.github/workflows/cross-browser-tests.yml#L12

Reply:
Got it. There was a similar check in place in the script but it feels more appropriate in the YAML config. PTAL #33707

Reply:
Perfect! Approved.